toloka_provider/hooks/toloka.py

```diff
@@ -32,18 +32,17 @@ def get_conn(self) -> TolokaClient:
         if not self.client:
             self.log.debug('Creating toloka client for conn_id: %s', self.toloka_conn_id)
 
-            extra_options = {}
             if not self.toloka_conn_id:
                 raise AirflowException('Failed to create toloka client. no toloka_conn_id provided')
 
             conn = self.get_connection(self.toloka_conn_id)
+
+            extra_options = {'env': 'PRODUCTION'}
             if conn.extra is not None:
                 extra_options = conn.extra_dejson
 
                 if 'env' in extra_options and extra_options['env'].upper() == 'SANDBOX':
                     extra_options['env'] = 'SANDBOX'
```

Comment:
It's better not to check the value env value here. Imagine you mistyped 'sandbox' to 'sadnbox'. Current code would ignore your `env`, output no warnings and run user's code right on production. I guess it's better to pass env value (if present) without additional checks:

```Python
extra_options = {} if conn.extra is None else conn.extra_dejson
etra_options['env'] = extra_options.get('env', 'PRODUCTION').upper()    
```
or maybe even without `upper()`
```Python
extra_options = {} if conn.extra is None else conn.extra_dejson
etra_options.setdefault('env', 'PRODUCTION')
```